cdap-spark-core3_2.12/src/k8s/Dockerfile

```diff
@@ -43,6 +43,12 @@ USER root
 RUN apt-get update && \
   apt-get -y install curl && \
   curl -L -o /opt/cdap/cdap-spark-core/lib/gcs-connector-hadoop2-latest.jar https://storage.googleapis.com/hadoop-lib/gcs/gcs-connector-hadoop2-latest.jar && \
```

Comment:
not related to s3, but while we're adding hdfs connectors, can you change this so that it pulls an exact version of this instead of picking the latest? I think we pick up hadoop2-2.2.6 right now.

Reply:
actually, let's use hadoop2-2.2.5 to make it consistent with the other image.

Reply:
👍 will make the change